website_logo/controllers/main.py

```diff
@@ -80,5 +80,5 @@ def website_logo(self, dbname=None, **kw):
                             image, filename=imgname, mtime=mtime)
                         return response
             except Exception:
-                return http.send_file(placeholder(imgname))
+                pass
```

Comment:
Using [pep-0020](https://www.python.org/dev/peps/pep-0020):
 - Errors should never pass silently.

If there’s really nothing to do add a `_logger.error`